packages/expo-cli/src/credentials/views/IosDistCert.ts

```diff
@@ -166,61 +193,257 @@ export class UpdateIosDist implements IView {
   async provideOrGenerate(ctx: Context): Promise<DistCert> {
     const userProvided = await promptForDistCert(ctx);
     if (userProvided) {
-      return userProvided;
+      if (!ctx.hasAppleCtx()) {
+        log(
+          "WARNING! Unable to validate Distribution Certificate due to insufficient Apple Credentials. Please double check that you're uploading valid files for your app otherwise you may encounter strange errors!"
```

Comment:
```suggestion
          chalk.yellow("WARNING! Unable to validate Distribution Certificate due to insufficient Apple Credentials. Please double check that you're uploading valid files for your app otherwise you may encounter strange errors!")
```